gateways/adyen/request_builder.go

```diff
@@ -60,6 +60,10 @@ func buildAuthRequest(authRequest *sleet.AuthorizationRequest, merchantAccount s
 		},
 		MerchantAccount:        merchantAccount,
 		MerchantOrderReference: authRequest.MerchantOrderReference,
+
+		// reference to uniquely identify shopper
+		// https://docs.adyen.com/api-explorer/#/CheckoutService/latest/payments__reqParam_shopperReference
+		ShopperReference: *authRequest.ClientTransactionReference,
```

Comment:
I think ClientTransactionReference will be unique to each transaction rather than to each shopper. Should we use some new reference since we are also using ClientTransactionReference on line 53?

Reply:
Created UUID as ShooperReference